content/docs/04-clusters/03-edge/04-edgeforge-workflow/12-build-artifacts.md

```diff
@@ -0,0 +1,309 @@
+
+  ```bash
+  export PROJECT_ID=[USE-YOUR-PROJECT-ID_HERE]
+  ```
+
+
+18. Build the offline content bundle using the following command. The content bundle will have the configuration and settings for the Edge cluster, including the operating system image, Kubernetes distribution, any additional packs, and manifest files specified in your cluster profile you wish to deploy to the Edge cluster.
+
+  The command below uses the palette-edge tool to build the offline content bundle. The `--api-key`, `--project-id`, and `--cluster-profile-ids` options specify the necessary parameters for the build process. The `--palette-endpoint` option specifies the endpoint URL for the Spectro Cloud API. The `--outfile` option specifies the name of the output file for the content bundle.
+<br />
+
+  ```bash
+  palette-edge build --api-key $API_KEY \
+  --project-id $PROJECT_ID \
+  --cluster-profile-ids $PROFILE_ID \
+  --palette-endpoint api.spectrocloud.com \
+  --outfile content
+  ```
+
+
+19. Use the command below to list all files in the current directory, including the newly built content bundle. The content bundle will have the following naming convention, `content-[randon-string]`, for example, **content-8e61a9e5**. The command below will verify that you successfully created the content bundle with the specified name.
+<br />
+
+  ```bash
+  ls -al
+  ```
+
+
+20. List the details of the content bundle using the following command. Replace the `content-[randon-string]` with the content bundle in your current directory. The command below will provide information such as file size, permissions, and modification date.
+<br />
+
+  ```bash
+  ls -l content-[randon-string]
+  ```
+
+
+21. Issue the following command to execute the **earthly.sh** script with elevated privileges. The `+iso` option specifies the build target. This command will generate an ISO image from the content bundle and other configurations you have specified in the **.arg** and **user-data** files. 
+<br />
+
+  ```bash
+  sudo ./earthly.sh +iso
+  ```
+
+
+22. List the Edge installer ISO image and checksum by issuing the following command from the **CanvOS/** directory.
+<br />
+
+  ```shell
+  ls build/
+  ```
+
+  ```shell hideClipboard
+  # Output
+  palette-edge-installer.iso      
+  palette-edge-installer.iso.sha256
+  ```
+
+# Validate
+
+To validate the Edge installer ISO image, you can follow these steps:
+<br />
+
+1. Create a bootable USB flash drive using any third-party software.
+
+
+2. Select a physical or virtual host machine to emulate as an edge device. Enable (Dynamic Host Configuration Protocol) DHCP on the edge device before proceeding with the installation process. Enabling DHCP is necessary for the device to obtain an IP address automatically from the network. 
+
+
+3. Use the software to flash a bare host machine with a bootable USB drive. Most software that creates a bootable USB drive will automatically validate the ISO image during the installation process. 
```

Comment:
Add a Next Steps that provides a summary and guides users to check out the following resources. 

[Deploy an Edge Cluster on VMware](https://deploy-preview-1400--docs-spectrocloud.netlify.app/clusters/edge/site-deployment/deploy-cluster)
[Prepare Edge Host for Installation](https://deploy-preview-1400--docs-spectrocloud.netlify.app/clusters/edge/site-deployment/stage)